p2p/simulations/network.go

```diff
@@ -589,14 +605,6 @@ func (self *Network) getConn(oneID, otherID discover.NodeID) *Conn {
 }
 
 func (self *Network) Shutdown() {
-	// disconnect all nodes
```

Comment:
This is left over from when stopping sim nodes didn't actually drop connections. Sim nodes now drop connections when stopped as they use a p2p server, so we don't need this.